chapter_linear-networks/linear-regression-concise.md

```diff
@@ -406,10 +406,22 @@ print('b的估计误差：', true_b - b)
 
 ## 练习
 
-1. 如果将小批量的总损失替换为小批量损失的平均值，你需要如何更改学习率？
-1. 查看深度学习框架文档，它们提供了哪些损失函数和初始化方法？用Huber损失代替原损失，即
+:begin_tab:`mxnet`
+1. 如果我们用`l = loss(output, y).mean()`代替`l = loss(output, y)`，我们需要把`trainer.step(batch_size)`改为`trainer.step(1)`，代码才会表现得相同。为什么？
+2. 查看MXNet文档，看看`gluon.loss`和`init`模块中提供了哪些损失函数和初始化方法。用Huber's loss代替损失。
+3. 如何访问`dense.weight`的梯度？
```

Comment:
We don't need "1. 2. 3.", but just list "1. 1. 1.", our book will auto compile to "1. 2. 3.". Could you change them back? Thanks